### PR TITLE
PSMDB-1535 change the warning message in assert

### PR DESCRIPTION
**Pull request by olexandr-havryliak**
See https://github.com/percona/percona-server-mongodb/commit/b890af7de045d1d91b01f90a453d81fc6631b29d

### CODE REVIEW

File: jstests/percona/noPassthroughWithMongod/validate_interrupt.js [hunk 1]
```diff
@@ -37,9 +37,9 @@
         // appear once or twice.
         assert((numWarnings === 1) || (numWarnings === 2),
                'Expected 1 or 2 validation warnings:\n' + tojson(res));
-        assert(res.warnings[0].includes('EBUSY'), 'Expected an EBUSY warning:\n' + tojson(res));
+        assert(res.warnings[0].includes('This is a transient issue as the collection was actively in use by other operations'), 'Expected an EBUSY warning:\n' + tojson(res));
         if (numWarnings === 2) {
-            assert(res.warnings[1].includes('EBUSY'), 'Expected an EBUSY warning:\n' + tojson(res));
+            assert(res.warnings[1].includes('This is a transient issue as the collection was actively in use by other operations'), 'Expected an EBUSY warning:\n' + tojson(res));
```
Comment (ktrushin) [inline jstests/percona/noPassthroughWithMongod/validate_interrupt.js:42]: For the purpose of completeness and clarity, I would also add the `Could not complete validation of <table_name>` sentence that precedes the `This is transient issue...` in the warning message. That will probably require using a regular expression to mask the table name.

Comment (olexandr-havryliak) [reply jstests/percona/noPassthroughWithMongod/validate_interrupt.js]: Changed to regex

**Review by ktrushin — APPROVED** [review]
Looks good!

**Review by ktrushin — APPROVED** [review]
Looks good!

**Pull request merged**